tezos/2-channel-establishment.md

```diff
@@ -71,11 +71,14 @@ The protocol proceeds as follows:
 
-      c.  They fund the contract by calling the [`addCustFunding` entrypoint](5-tezos-escrowagent.md#addcustfunding-entrypoint) of the contract. The source of this transfer operation must be the `customer_address` specified in the contract's initial storage and the transfer amount must be equal to `init_customer_balance`. When the chain watcher indicates that the `addCustFunding` operation group is confirmed to the depth `required_confirmations`, they update the channel status to `CustomerFunded`.
-      
-6. The customer sends the [`funding_confirmed` message](#the-funding_confirmed-message) to the merchant, which contains the contract identifier `contract-id`.
-7. The merchant [checks the corresponding contract and initial storage for the expected values](#merchant-requirements-4) and initializes the chain watcher to track the zkChannels contract specified by `contract_ID`. The merchant then funds their side of the smart contract, if the channel is dual-funded, by calling the [`AddMerchFunding` entrypoint](5-tezos-escrowagent.md#addmerchfunding-entrypoint) of the contract with identifier `contract_id`. When the chain watcher indicates  this contract has status `OPEN` for a depth of `required_confirmations` blocks, the merchant runs `zkAbacus.Activate()` to generate the initial payment tag and sends the customer [the message `activate`](#the-activate-message), which contains this payment tag. 
-8. Upon completion of `zkAbacus.Activate()`, the zkChannel is open and ready for [payments](3-channel-payments.md). 
+      c. They send the contract identifer `contract_ID` to the merchant.
+
+6.  The merchant [checks the corresponding contract and initial storage for the expected values](#merchant-requirements-4) and initializes the chain watcher to track the zkChannels contract specified by `contract_ID`. If the values are not as expected, the merchant aborts.
+
+7. The customer funds the [zkChannels contract](5-tezos-escrowagent#zkchannels-contract) by calling the [`addCustFunding` entrypoint](5-tezos-escrowagent.md#addcustfunding-entrypoint) of the contract. The source of this transfer operation must be the `customer_address` specified in the contract's initial storage and the transfer amount must be equal to `init_customer_balance`. When the chain watcher indicates that the `addCustFunding` operation group is confirmed to the depth `required_confirmations`, they update the channel status to `CustomerFunded`. The customer then sends the [`funding_confirmed` message](#the-funding_confirmed-message) to the merchant, which contains the contract identifier `contract-id`.
```

Comment:
I think the last line is not quite right any more. The `funding_confirmed` message is now just a ping with no content. I would say something like,

"The customer then sends a bit to the merchant, indicating that their funding is confirmed on chain."

Reply:
Oh, actually I see you updated the message description below, so I would just remove the clause here about it containing the `contract-id`.